int8 battery_percent

string flying_state
string flying_mode
string plane_gear_box

int8 gun_id
string gun_state
int8 claw_id
string claw_state